packages/algob/src/internal/core/config/config-validation.ts

```diff
@@ -156,7 +159,12 @@ export function getValidationErrors(config: any): CfgErrors {
 				errors.push(net, "host", host, "hostname string (eg: http://example.com)");
 			}
 			const token = hcfg.token;
-			if (typeof token === "string" && token.length < 10) {
+			if (
+				typeof token === "string" &&
+				token.length < 10 &&
+				host !== "https://node.testnet.algoexplorerapi.io" &&
+				host !== "https://node.algoexplorerapi.io"
```

Comment:
Host can not start with HTTP because it would cause an error
![image](https://user-images.githubusercontent.com/24353881/161715057-ed80b328-113a-4107-bb5b-05d3c161c288.png)